### PR TITLE
Add "Drinks" Card

### PR DESCRIPTION
**Pull request by skade**
Here's a chapter for the "drinks" feature.

### CODE REVIEW

File: cards/drinks.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+# Drinks
+
+## Description
+
+Non-alcoholic drinks are a base necessity of every conference. Luckily, they are also quite easy to organise.
+
+## Categorisation
+
+Type: catering
```
Comment (celaus) [inline cards/drinks.md]: Add an empty lines or enumeration to keep the formatting

File: cards/drinks.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+While you can skip food on conferences, you can't skip drinks on any event that is longer then ~1.5 hours. You don't want people to have to step out to quench their thirst.
+
+Basic drinks include water, some soft drinks and maybe tea/coffee. Also try to find drinks with taste but low (or no) amount of sugar, not everyone likes or can consume large amounts of sugar.
+
+Plan with 3 liters of water per person, per day. Especially in times where it can _suddenly_ get hot, having ample amount of water is important! When it gets hot, people will start switching from enjoyable drinks to lots of water.
+
+## How?
+
+Drinks can be easily bought from delivery services. Usually, even small numbers can be covered by a delivery service.
```
Comment (celaus) [inline cards/drinks.md]: Bottles vs cups vs glass?

File: cards/drinks.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+## Tasks and Time Frames
+
+### Check the venue details
+
+* Is there a fridge?
+* If not, can you bring one?
+* Does the venue offer drinks?
+  * If yes, will they insist to take their offer?
+
+When? Before signing the venue.
```
Comment (celaus) [inline cards/drinks.md]: + newline

File: cards/drinks.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+Suggestion: Roughly 6 months before the event
+
+### Get quote
+
+Get a proper quote from the suppliers you picked.
+
+When? When you have a rough idea of how many people will attend.
```
Comment (celaus) [inline cards/drinks.md]: + newline

File: cards/drinks.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+Suggestion: About 1 month before the event.
+
+### Optional: Plan service schedule
+
+Should you decide to service drinks yourselves, come up with a service schedule for the conference. Who fills the fridges, how often?
+
+When? Right before the conf.
```
Comment (celaus) [inline cards/drinks.md]: +newline

File: cards/drinks.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+Suggestion: 1 week to 1 day before the conference.
+
+### Pay up
+
+Don't forget paying the final invoice after the event.
+
+When? After the conf.
```
Comment (celaus) [inline cards/drinks.md]: +newline

**Comment by skade** [comment]
Added the fixes and a comment about bottled vs. not.

**Comment by celaus** [comment]
LGTM :+1:

**Comment by skade** [comment]
@celaus That should be fixed now. Thanks for the review!

**Pull request merged**